map/map_projection_loader/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>map_projection_loader</name>
+  <version>0.1.0</version>
+  <description>map_projection_loader package as a ROS 2 node</description>
+  <maintainer email="koji.minoda@tier4.jp">Koji Minoda</maintainer>
```

Comment:
@kminoda Can another person, in addition to you, be assigned as a maintainer?

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/3986/commits/378d9dc8ae70613bd382f248e88f04a56a1a4197